rust/src/detect/parser.rs

```diff
@@ -22,12 +22,28 @@ use nom7::character::complete::multispace0;
 use nom7::sequence::preceded;
 use nom7::IResult;
 
+#[derive(Debug)]
+pub enum ResultValue {
+    Numeric(u64),
+    String(String),
+}
+
 static WHITESPACE: &str = " \t\r\n";
 /// Parse all characters up until the next whitespace character.
 pub fn take_until_whitespace(input: &str) -> IResult<&str, &str, RuleParseError<&str>> {
     nom7::bytes::complete::is_not(WHITESPACE)(input)
 }
 
+// Parsed as a u64 for validation with u32 {min,max} so values greater than uint32
```

Comment:
I do not see the code matching the comment for u32 min max... Does it ?

Reply:
Good catch; I'll update the comment.